docs/ocs-on-a-disconnected-environment.md

```diff
@@ -53,129 +53,148 @@ you should get something similar to this:
+  registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:faeb96365fafba1255ca3b9ad7e3f6ddd8791588b9ff283a7acffd7d5c89fe09=<your_registry>/openshift4/ose-csi-external-attacher
+  registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:0570eacd3cb8b5d60fd3ecf4c80860280725c81d83bc2a04326d3393c6dcb2be=<your_registry>/openshift4/ose-csi-external-provisioner-rhel7
+  registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:e7302652fe3f698f8211742d08b2dcea9d77925de458eb30c20789e12ee7ae33=<your_registry>/openshift4/ose-csi-external-resizer-rhel7
+  registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:254c9046eaf61c451b40bac3f897c40bc59f187e5313b9799fbcacf17398a191=<your_registry>/rhscl/mongodb-36-rhel7
+  quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=<your_registry>/noobaa/pause
+  registry.redhat.io/ocs4/ocs-must-gather-rhel8@sha256:a512e88410cadd6e4b286aca63b8917389dd73e8c202189563055c37f4c50fef
+  ```
+
+  in OCS-4.4 the relatedImages do appear in the CSV, but we still need to fix the missing pause image for lib-bucket-provisioner.  
+  Also the `ose-csi-external-resizer` image is missing in the relatedImage (Although it is not yet used by OCS we'll mirror it anyway)
+
+  mapping.txt for OCS-4.4.0:
+  ```
+  registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:e7302652fe3f698f8211742d08b2dcea9d77925de458eb30c20789e12ee7ae33=<your_registry>/openshift4/ose-csi-external-resizer-rhel7
+  quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=<your_registry>/noobaa/pause
+  registry.redhat.io/ocs4/ocs-must-gather-rhel8@sha256:823e0fb90bb272997746eb4923463cef597cc74818cd9050f791b64df4f2c9b2
+  ```
 
 - **Mirror the images in `mapping.txt`**  
-```
-oc image mirror -f mapping.txt --insecure --registry-config=${AUTH_FILE}
-```
+  ```
+  oc image mirror -f mapping.txt --insecure --registry-config=${AUTH_FILE}
+  ```
+
+
+- **Add the images we mirrored in `mapping.txt` to the imageContentSourcePolicy**  
+After `oc adm catalog mirror` is completed it will print an output dir where an `imageContentSourcePolicy.yaml` is generated (usually the catalog image with `-manifests` suffix. e.g: `redhat-operators-manifests`).
+the missing related images should be added to the `ImageContentSourcePolicy` spec. e.g. an entry for rhceph-rhel8: 
+  ```yaml
+    - mirrors:
+      - <your_registry>/ocs4/rhceph-rhel8
+      source: registry.redhat.io/ocs4/rhceph-rhel8
+  ```
+
+
+- **After adding the related images (required for OCS-4.3 only), apply the `imageContentSourcePolicy.yaml` file to the cluster**  
+(the file is generated by `oc adm catalog mirror`. the output dir is usually `./[catalog image name]-manifests`)
+  ```
+  oc apply -f ./[output dir]/imageContentSourcePolicy.yaml
+  ```
```

Comment:
The `oc apply ...` command with `imageContentSourcePolicy.yaml` file is not a workaround so it should be outside of workaround section.